scribus/text/storytext.cpp

```diff
@@ -97,6 +97,9 @@ StoryText::StoryText(const StoryText & other) : QObject(), SaxIO(), m_doc(other.
 	invalidateLayout();
 }
 
+UErrorCode StoryText::status = U_ZERO_ERROR;
+BreakIterator* StoryText::m_WordBreakIterator = BreakIterator::createWordInstance(NULL, status);
```

Comment:
You are not checking the value of status to see if creating the iterator failed or not. I suggest instead to set the iterator to NULL here and in places where you use it, create it if it is NULL.